packages/stores/src/stores/ExchangeRatesStore.ts

```diff
@@ -0,0 +1,8 @@
+import type { ExchangeRatesResponse } from '@deriv/api-types';
+import BaseStore from './BaseStore';
+
+export default class ExchangeRatesStore extends BaseStore<ExchangeRatesResponse['exchange_rates']> {
```

Comment:
Do we need a store just for exchange rates?

Reply:
@balakrishna-deriv Yes 👀

- This state will be used all throughout the application, So we can keep the data in state management and make it accessible for everyone, Later we will do the same things for other similar things like `balance` or `website_status`.
- We need to move our stores states from `@deriv/core` to `@deriv/stores` which is very hard to do, So what we can do for now is for new things we can create the stores in the `@deriv/stores` package and then slowly remove the legacy code from old stores.
- State management should only keep the state and change it via an action, We shouldn't put business logic here. For that, we will be using hooks that just consume the stored data.
- Instead of having big stores to keep many states as we have, We can have small stores with single responsibility to keep track of the data flow. Maybe someday we can have atomic state management.
- This would also make it easier for us to change the state management library in the future if needed as the change will impact only a small part of the code each time